src/components/CreateOrUpdateRequest.svelte

```diff
@@ -29,6 +29,7 @@ $: isNew = !request.id
 $: if ($me.organizations && $me.organizations.length > 0) {
   request.viewableBy = $me.organizations[0].id
 }
+$: neededBefore = request.needed_before && request.needed_before.split("T")[0] || ''
```

Comment:
would `$: neededBefore = request.needed_before?.split("T")[0] || ''` work? I find I am sometimes wrong just simulating it in my head.

Reply:
I was about to say I tried that, but I hadn't tried it on the `.` preceding `split`. I tested in the console and it seems to work.